### DISCUSSION TITLE
Cross platform configuration tool

### DISCUSSION THREAD

**Discussion by csutcliff** [danielnilsson9/bbs-fw#188] [Ideas]
Hi @danielnilsson9,

I'm most of the way through porting the configuration tool from WPF to [Avalonia](https://www.avaloniaui.net/) in order to be cross-platform (Linux in my case however it should work for MacOS too).

Predominantly I'm doing this for my own gain (I don't have a Windows machine) and as a relatively simple first attempt at WPF -> Avalonia porting however I wondered if there would be any interest in up-streaming the result once I have completed and tested it?

I'll reply back once I've got a first pass complete with a link to my branch.

**Comment by DavesDevFails** [comment]
Never heard of Avalonia, but I would def be interested in an android version (not sure how simple it is to "create one") using Avalonia once you have ported the config tool to it

**Comment by skeeterman007** [comment]
> Never heard of Avalonia, but I would def be interested in an android version (not sure how simple it is to "create one") using Avalonia once you have ported the config tool to it

yeah or if there was a way to connect to the controller and configure it via bluetooth from your phone that would be great.

**Reply by DavesDevFails** [reply]
Bluetooth would be ideal, but I'm sure we would need some extra hardware for that.
